client/my-sites/migrate/step-upgrade.jsx

```diff
@@ -144,6 +144,7 @@ class StepUpgrade extends Component {
 							</div>
 						</div>
 					</div>
+					{ /* TODO: is the following "import everything" behaviour up to date? */ }
```

Comment:
Is the following "import everything" behaviour up to date? I don't have enough context to understand if it can stay as-is, or if it can be deleted, or otherwise tweaked.